eu_einvoice/european_e_invoice/custom/sales_invoice.py

```diff
@@ -794,6 +874,9 @@ def attach_xml_to_pdf(invoice_id: str, pdf_data: bytes) -> bytes:
 	"""
 	from drafthorse.pdf import attach_xml
 
+	if _is_ghostscript_installed():
+		pdf_data = _convert_pdf_to_pdfa(pdf_data)
```

Comment:
```suggestion
		try:
			pdf_data = _convert_pdf_to_pdfa(pdf_data)
		except RuntimeError:
			frappe.log_error()
```

This way we can still write invoices when Ghostscript is installed but fails, while at the same time alerting the System Manager that something is off.